OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -186,6 +194,15 @@ private void SpeedRunButton_OnClick()
             _justEnteredGame = true;
             GameObject.Find(RESUME_BUTTON_NAME).GetComponent<SubmitActionLoadScene>().Submit();
         }
+        
+        private void ResetRunButton_OnClick()
+        {
+            _modEnabled = true;
+            _justEnteredGame = true;
+            ResetSpawnNames();
```

Comment:
This should also not be necessary; ResetSpawnNames() already gets called during Update() when _justEnteredGame is true.

Reply:
gotcha